code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -506,115 +519,403 @@
 	reagent_state = LIQUID
-	color = "#FF80FF"
+	color = "#888888"
 	metabolism = 0.01
 	data = 0
-
-/datum/reagent/citalopram/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
-	if(volume <= 0.1 && data != -1)
-		data = -1
-		M << "<span class='warning'>Your mind feels a little less stable...</span>"
-		if (ishuman(M))
-			var/mob/living/carbon/human/H = M
-			var/obj/item/organ/brain/B = H.internal_organs_by_name["brain"]
-			if(B)
-				for(var/x in B.traumas)
-					var/datum/brain_trauma/BT = x
-					if((istype(BT, BRAIN_TRAUMA_MILD) || istype(BT, BRAIN_TRAUMA_SPECIAL)) && BT.suppressed)
-						BT.on_gain()
-						BT.suppressed = 0
-
-	else
-		if((world.time > data + ANTIDEPRESSANT_MESSAGE_DELAY) || !data)
-			data = world.time
-			M << "<span class='notice'>Your mind feels stable... a little stable.</span>"
-
-			if (ishuman(M))
-				var/mob/living/carbon/human/H = M
-				var/obj/item/organ/brain/B = H.internal_organs_by_name["brain"]
-				if(B)
-					for(var/x in B.traumas)
-						var/datum/brain_trauma/BT = x
-						if((istype(BT, BRAIN_TRAUMA_MILD) || istype(BT, BRAIN_TRAUMA_SPECIAL)) && !BT.permanent)
-							BT.on_lose()
-							BT.suppressed = 1
-
-/datum/reagent/paroxetine
-	name = "Paroxetine"
+	taste_description = "paper"
+	goodmessage = list("You feel focused.","You feel like you have no distractions.","You feel willing to work.")
+	badmessage = list("You feel a little distracted...","You feel slight agitation...","You feel a dislike towards work...")
+	worstmessage = list("You feel completely distrtacted...","You feel like you don't want to work...","You think you see things...")
+	cure_effects = list(
+		/datum/brain_trauma/special/imaginary_friend = 40,
+		/datum/brain_trauma/mild/hallucinations = 20,
+		/datum/brain_trauma/mild/phobia/ = 20
+	)
+	dosage_effects = list(
+		/datum/brain_trauma/mild/hallucinations = 5
+	)
+	withdraw_effects = list(
+		/datum/brain_trauma/mild/phobia/ = 5,
+		/datum/brain_trauma/mild/hallucinations = 2
+	)
+
+/datum/reagent/antidepressants/fluvoxamine
+	name = "fluvoxamine"
+	id = "fluvoxamine"
+	description = "Fluvoxamine is safe and effective at treating basic phobias, as well as schizophrenia and muscle weakness at higher doses. Withdrawl effects are rare. Side effects are rare, and include hallucinations."
+	reagent_state = LIQUID
+	color = "#888888"
+	metabolism = 0.01
+	data = 0
+	taste_description = "paper"
+	goodmessage = list("You do not feel the need to worry about simple things.","You feel calm and level-headed.","You feel fine.")
+	badmessage = list("You feel a little blue.","You feel slight agitation...","You feel a little nervous...")
+	worstmessage = list("You worry about the littlest thing...","You feel like you are at risk...","You think you see things...")
+	cure_effects = list(
```

Comment:
Should also include hallucinations, since thats one of the dosage traumas.
Same thing with the other meds

Reply:
I'm not sure I quite follow. Dosage effects are base percent chance to add a trauma when the medication is taken.